pages/science/reactions.rst

```diff
@@ -344,6 +395,24 @@ Sticking reactions can be defined in the CTI format using the `stick` entry, or
 in the YAML format by specifying the rate constant in the reaction's
 :ref:`sticking-coefficient <sec-yaml-interface-reaction>` field.
 
+Surface Blowers-Masel Reactions
+-------------------------------
+
+.. TODO: Update the link once version 2.6 is released
+
+Surface Blowers-Masel Reactions have the same Arrhenius-like rate expression described in
+`Surface Reactions <https://cantera.org/science/reactions.html#surface-reactions>`__,
+and the activation energy :math:`E_a` is determined
+as described in `Blowers-Masel <https://cantera.org/documentation/dev/sphinx/html/yaml/reactions.html#sec-yaml-blowers-masel>`__.
```

Comment:
I think this should be a link to the Blowers-Masel documentation on this page, so you can use a `:ref:` link.